lib/replset.js

```diff
@@ -167,6 +200,7 @@ class ReplSet extends EventEmitter {
         var config = generateConfiguration(
           self.replSet,
           self.version,
+          protocolVersion,
```

Comment:
This is fine, but I wonder if it's actually useful to save off `protocolVersion` anymore, and why we aren't just passing the `options` right to `generateConfiguration`?

Reply:
I misunderstood this comment the first time. I just did it for 'cleaner code' in my opinion. I can give that a whirl and see how it looks though.